packages/kbn-alerts-as-data-utils/src/field_maps/alert_field_map.ts

```diff
@@ -155,6 +156,13 @@ export const alertFieldMap = {
     array: false,
     required: false,
   },
+  [ALERT_URL]: {
+    type: 'keyword',
+    array: false,
+    index: false,
+    required: false,
+    ignore_above: 2048,
+  },
```

Comment:
Are we intentionally making this field [doc values only](https://www.elastic.co/guide/en/elasticsearch/reference/current/doc-values.html#doc-value-only-fields)? What are the intended use cases for this field mapping?

Reply:
As we discussed, we are not going to search on this field, it is just for representation.